cmd/seq-coordinator-manager/seq-coordinator-manager.go

```diff
@@ -0,0 +1,308 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/enescakir/emoji"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/gdamore/tcell/v2"
+	"github.com/offchainlabs/nitro/cmd/seq-coordinator-manager/rediscoordinator"
+	"github.com/offchainlabs/nitro/util/redisutil"
+	"github.com/rivo/tview"
+)
+
+// Tview
+var pages = tview.NewPages()
+var app = tview.NewApplication()
+
+// Lists
+var prioritySeqList = tview.NewList().ShowSecondaryText(false)
+var nonPrioritySeqList = tview.NewList().ShowSecondaryText(false)
+
+// Forms
+var addSeqForm = tview.NewForm()
+var priorityForm = tview.NewForm()
+var nonPriorityForm = tview.NewForm()
+
+// Sequencer coordinator managment UI data store
+type manager struct {
+	redisCoordinator *rediscoordinator.RedisCoordinator
+	prioritiesMap    map[string]int
```

Comment:
prioritiesMap and livelinessMap seems to be being used as sets right? (please correct me if I am wrong)
If that's the case, let's use map[string]bool instead of map[string]int, that makes it clear that its a set and not a counting map. Also maybe rename prioritiesMap and livelinessMap  to prioritiesSet and livelinessSet.

Reply:
agreed (Y)